app/views/grade_entry_forms/_form.html.erb

```diff
@@ -89,3 +89,12 @@
                  data: { disable_with: t('working') } %>
   </p>
 <% end %>
+
+<% if @grade_entry_form.id != nil %>
+  <% deletable = @grade_entry_form.count_non_nil == 0 %>
+  <%= button_to t(:delete),
+                course_grade_entry_form_path(@current_course.id, @grade_entry_form.id),
+                method: :delete,
+                disabled: !deletable
```

Comment:
This is good, but please also add a tooltip with a message explains why the button is disabled (see Mimis' PR for assignment button deletion).